src/rulesets/LinearAlgebra/symmetric.jl

```diff
@@ -288,6 +454,18 @@ _symhermtype(::Type{<:Symmetric}) = Symmetric
 _symhermtype(::Type{<:Hermitian}) = Hermitian
 _symhermtype(A) = _symhermtype(typeof(A))
 
+function _realifydiag!(A)
+    for i in axes(A, 1)
```

Comment:
Why this and not `diaginds` ?
Is faster? or clearer?

Reply:
It was in an earlier benchmark, but I just double-checked, and I'm not seeing that now. I'll switch to `diagind`, it's safer.